api/rpc/account/account.proto

```diff
@@ -171,7 +173,7 @@ message AccountRequest {
   string name = 1;
 }
 
-message Account {
+message AccountInfo {
```

Comment:
Oops! You renamed it so not to conflict with the other Account message. Sorry, my bad. Let me chew on this a moment....